src/celeritas/phys/FourVector.hh

```diff
@@ -37,6 +56,33 @@ struct FourVector
 
 //---------------------------------------------------------------------------//
 // INLINE UTILITY FUNCTIONS
+//---------------------------------------------------------------------------//
+/*!
+ * Construct from rest mass, momentum, direction.
+ *
+ * Note that this uses \c std::hypot which yields more accurate answers if the
```

Comment:
Where is `std::hypot()` being called?

Reply:
It's not any more since it changed the results on some of the CI platforms... we should consider a platform-portable reimplementation for reproducibility.